BarcodeSDK.MAUI.Example/HomePage.xaml.cs

```diff
@@ -44,16 +48,17 @@ public HomePage()
         private void InitMenuItems()
         {
             MenuItems = [
-                new HomePageMenuItem("RTU - Single Scanning", SingleScanning),
-                new HomePageMenuItem("RTU - Single Scanning Selection Overlay", SingleScanningWithArOverlay),
-                new HomePageMenuItem("RTU - Batch Barcode Scanning", BatchBarcodeScanning),
-                new HomePageMenuItem("RTU - Multiple Unique Barcode Scanning", MultipleUniqueBarcodeScanning),
-                new HomePageMenuItem("RTU - Find and Pick Barcode Scanning", FindAndPickScanning),
+                new HomePageMenuItem("RTU - Single Scanning", SingleScanningFeature.StartSingleScanningAsync),
+                new HomePageMenuItem("RTU - Single Scanning Selection Overlay", SingleScanningWithArOverlayFeature.StartSingleScanningWithArOverlayAsync),
+                new HomePageMenuItem("RTU - Batch Barcode Scanning", BatchBarcodeScanningFeature.StartBatchBarcodeScanningAsync),
+                new HomePageMenuItem("RTU - Multiple Unique Barcode Scanning", MultipleUniqueBarcodeScanningFeature.StartMultipleUniqueBarcodeScanningAsync),
+                new HomePageMenuItem("RTU - Find and Pick Barcode Scanning", FindAndPickScanningFeature.StartFindAndPickScanningAsync),
                 new HomePageMenuItem("Classic Component - Barcode Scanning", () => Navigation.PushAsync(new BarcodeClassicComponentPage())),
+                new HomePageMenuItem("Classic Component - Barcode Scanning (MVVM)", () => Navigation.PushAsync(new BarcodeClassicComponentView())),
                 new HomePageMenuItem("Classic Component - Selection Overlay", () => Navigation.PushAsync(new BarcodeArOverlayClassicComponentPage())),
                 new HomePageMenuItem("Classic Component - Scan and Count", () => Navigation.PushAsync(new BarcodeScanAndCountClassicComponentPage())),
                 new HomePageMenuItem("Detect Barcodes on Image", DetectBarcodesOnImage),
```

Comment:
Inside this function, please use the `Microsoft.Maui.Essentials` ImagePicker. So in the upcoming release candidate, there will be no breaking change due to the `ImagePicker`, as we decided to remove it.

Refer: [ScanbotSdkExample.Maui](https://github.com/doo/scanbot-sdk-maui-example/blob/b78422b18554f9aa6d49f470f83939bb11593cca/MAUI/ScanbotSdkExample.Maui/Pages/HomePage.xaml.cs#L116-L142)